src/site/layouts/page-breadcrumbs.html

```diff
@@ -12,6 +12,25 @@
   {% include "src/site/includes/decision-reviews-banner.html" %}
 {% endif %}
 
+{% if banner %}
+  <div class="usa-alert-full-width usa-alert-full-width-warning" role="region">
```

Comment:
Let's add an `aria-labelledby="alert-heading-text"` here, to go with the `role="region"`. Having that content-driven name will give this banner more semantic meaning to SR's.